packages/ember-cli-fastboot/package.json

```diff
@@ -43,47 +49,61 @@
     "silent-error": "^1.1.0"
   },
   "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.2.5",
+    "@embroider/test-setup": "^0.37.0",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
+    "babel-eslint": "^10.1.0",
     "body-parser": "^1.18.3",
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-test-helper": "^1.5.0",
     "chai": "^4.1.2",
     "chai-fs": "^2.0.0",
     "chai-string": "^1.4.0",
-    "ember-ajax": "^3.1.0",
-    "ember-cli": "~3.3.0",
+    "ember-auto-import": "^1.11.2",
+    "ember-cli": "~3.26.1",
     "ember-cli-addon-tests": "^0.11.1",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.8.2",
-    "ember-cli-qunit": "^4.3.2",
-    "ember-cli-shims": "^1.2.0",
+    "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-terser": "^4.0.1",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
-    "ember-resolver": "^5.0.1",
+    "ember-page-title": "^6.2.1",
+    "ember-qunit": "^5.1.4",
+    "ember-resolver": "^8.0.2",
     "ember-sinon": "^2.2.0",
-    "ember-source": "~3.8.0",
-    "eslint-plugin-ember": "^7.0.0",
-    "eslint-plugin-node": "^7.0.1",
+    "ember-source": "~3.26.1",
+    "ember-template-lint": "^3.2.0",
+    "eslint": "^7.23.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-ember": "^10.3.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "glob": "^7.1.3",
     "loader.js": "^4.7.0",
     "mocha": "^5.2.0",
-    "qunit-dom": "^0.8.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.2.1",
+    "qunit": "^2.14.1",
+    "qunit-dom": "^1.6.0",
     "release-it": "^12.0.1",
     "release-it-lerna-changelog": "^1.0.2",
     "request": "^2.88.0",
     "rsvp": "^4.8.3"
   },
   "engines": {
-    "node": "10.* || 12.* || >= 14.*"
+    "node": "10.* || >= 12"
```

Comment:
Should we update this to node 12 and later since we are still pre 3.0?  OR would you want that as a separate PR?

Reply:
Happy to contribute if so

Reply:
So... I was thinking it would be better to do them in a different PR. This PR was to try and fix the odd behaviour in these tests: https://github.com/ember-fastboot/ember-cli-fastboot/pull/824

The more I think about it I think we need to update with ember-cli-update and change the node requirements in the same PR

Reply:
Seems sane!

Reply:
I ended up needing to pull in the engine requirement changes in this PR anyway 👍